modules/imgimport/include/csvreading.h

```diff
@@ -0,0 +1,22 @@
+#ifndef CSV_READING_H_INCLUDED
+#define CSV_READING_H_INCLUDED
+
+#include "metadata.h";
+using namespace std;
+
+struct Metadatalinkedlist{
+    Metadata a;
+    Metadatalinkedlist* next;
+};
+
+/**
+* @brief read a csv file creates a linked list of metadata
```

Comment:
Just a suggestion here, but it might be prudent to be more specific as to the exact input read, i.e. that this is for reading telemetry logs from the plane.
Also, considering that you are handing file i/o, it might be worth noting how the function handles, for example, the file not existing, or other unusual situations.